Automattic/wp-calypso
# Happy Blocks: add new block

## Proposed Changes

This PR moves a few things around within Happy Blocks and adds a new block `universal-header`. This is a block for the WordPress.com header navigation. This will allow us to use it in all our other themes on WordPress.com as well as Calypso where it originally lived.

## Testing Instructions

1. Pull branch and `yarn && cd apps/happy-blocks && yarn dev --sync`
2. Sandbox WordPress.com/support and WordPress.com/forums
3. Create a new topic in forums and verify the upgrade blocks still work with no regression.
4. [Go to this PR for the testing steps on WordPress.com/support](247-gh-Automattic/wpsupport3)
